src/bogart/AS_BAT_MarkRepeatReads.C

```diff
@@ -791,24 +634,21 @@ findConfusedEdges(TigVector            &tigs,
         continue;               //  contained in the region.  Skip!
       }
 
-#if 0
-      if ((rMin  <= rdAlo) &&   //  If the read is contained in the repeat region,
-          (rdAhi <= rMax)) {    //  it's useless for deciding if this is a
-        continue;               //  confused repeat.
-      }
-#endif
+      //  This read intersects this repeat region.  Find the reads we used to
+      //  construct the tig originally.
+      //
+      //  Generate a log that we're checking this guy for confusion.  The log
+      //  will be reported only if we find a thickest edge.
 
-      //  This read intersects this repeat region.  Find the
-      //  reads we used to construct the tig originally.
+      char *logMsg = new char [1024];
```

Comment:
What's going on with logMsg???

Reply:
I needed a way of outputting a message if either of the following functions will do something,  be silent if they will do nothing, and output it only once.

Reply:
And they didn't have a way of forming the message themselves? Or the crucial point was about outputting once?